packages/terra-date-picker/src/DatePicker.module.scss

```diff
@@ -1,82 +1,240 @@
-/* stylelint-disable selector-max-compound-selectors, max-nesting-depth */
+// stylelint-disable selector-max-compound-selectors, max-nesting-depth
+// stylelint-disable declaration-property-value-blacklist scss/at-mixin-pattern
+// stylelint-disable scss/at-mixin-pattern
+
 :local {
-  .date-picker {
+  @mixin focus-styles {
+    border-color: var(--terra-date-picker-focus-border-color, #26a2e5);
+    border-style: var(--terra-date-picker-focus-border-style, solid);
+    border-width: var(--terra-date-picker-focus-border-width, 1px);
+    opacity: var(--terra-date-picker-focus-opacity, 1);
+    outline: none;
+  }
+
+  .date-input-container {
+    display: inline-block;
+  }
+
+  .date-input {
+    background-color: var(--terra-date-picker-background-color, #fff);
```

Comment:
@neilpfeiffer @jeremyfuksa  Should we be adding new theme variables with these changes? Are there callouts to reference. I'm worried we are adding in vars too early and increasing the complexity of the themes.

Reply:
Callouts haven't updated since 2017: https://zpl.io/ag0J0lO

@neilpfeiffer, does Seth have something mocked up?